### ISSUE TITLE
Playlists are gone after update to RC9

### ISSUE THREAD

**Issue by Arjanvde** [athombv/homey#1397]
**What is your Homey version (Settings → System)?**

1.2.0 RC9

**What did you try to achieve?**

After updating to RC9 all songs added to the favorite playlist and all created playlists (including songs) are gone.

**What was the result?**

Playlists are gone.

**What did you expect as result?**

Playlisits to be there.

**Comment by Arjanvde** [comment]
@TheMSB I might be able to check this with a second Homey (still running RC8) if this is fixed quickly.

**Comment by TheMSB** [comment]
@Arjanvde I am wondering what the cause could be right now. If I remember correctly you had difficulties creating playlists on RC8 did you not? I am thinking that it might be related.

**Comment by Arjanvde** [comment]
@TheMSB Yes I had, can try to update the second Homey and see if the playlists are deleted on that one as well?

**Comment by caseda** [comment]
I had no troubles creating playlist(s), but can confirm that playlists are deleted.

even though i only had 1 with 1 song in it

**Comment by TheMSB** [comment]
@Arjanvde 

Although I am not certain about the direct cause of the issue I have a potential fix, If you have not yet updated your second Homey I would like to request you wait with that until the next RC.

**Comment by Arjanvde** [comment]
@TheMSB 

Unfortunately that Homey did the update by itself last night. The songs that where added to the playlists where deleted, all other playlists did stay intact.

**Comment by chrisboer** [comment]
Same issue here with the playlists deleted
